app/views/stats/index.phtml

```diff
@@ -95,39 +95,102 @@
 				?></script>
 			</div>
 		</div>
+		
+		<?php
+// Function to generate a color palette
+function generateColorPalette($count) {
+    $colors = [];
+    for ($i = 0; $i < $count; $i++) {
+        // Generate visually distinct colors using HSV to RGB conversion
+        $hue = ($i / $count) * 360; // Distribute colors evenly around the color wheel
+        $saturation = 70; // Fixed saturation
+        $lightness = 50; // Fixed lightness
+        $colors[] = hslToHex($hue, $saturation, $lightness);
```

Comment:
Does the stats pie-chart script require hexadecimal colors? CSS supports `hsl()` natively for a very long time now, so this feels a bit redundant.

https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/hsl

Reply:
Yes, thank you for pointing this out.